python/HeaderService/hutils.py

```diff
@@ -384,17 +275,23 @@ def write_headers(self, filenames, MP=False, NP=2, md5=False):
                 write_header_string(arg)
         return
 
-    def write_header_emptyHDU(self, filename):
-
+    def write_header_fits(self, filename):
+        """Write a header file using the FITS format with empty HDUs"""
         data = None
         with fitsio.FITS(filename, 'rw', clobber=True, ignore_empty=True) as fits:
             for extname in self.HDRLIST:
                 hdr = copy.deepcopy(self.header[extname])
                 fits.write(data, header=hdr, extname=extname)
 
+    def write_header_string(self, filename):
+        """ Write header as string"""
+        self.string_header()
+        with open(filename, 'w') as fobj:
+            fobj.write(self.hstring)
```

Comment:
I have a slight preference for `print(self.hstring, file=fobj, end='')` as being more expressive, but `write()` is allowed.

Reply:
will keep the `write()` method as I prefer it.